localisation/00_wc_wonders_upgrades.csv

```diff
@@ -48,6 +48,32 @@ upgrade_silithid_greathive;Great Silithid Hive;Grande ruche silithide;Gro
 upgrade_silithid_greathive_desc;The great silithid hive located in Ahn'Qiraj houses the biggest of their queens who is constantly laying eggs to produce armies of inscetoid warriors.;La grande ruche silithide d'Ahn'Qiraj abrite la plus grande de leurs reines, qui pond constamment des �ufs pour produire des arm�es de guerriers insc�to�des.;Der gro�e Silithiden-Schwarmstock in Ahn'Qiraj beherbergt die gr��te ihrer K�niginnen, die unerm�dlich Eier legt, um Armeen insektoider Krieger zu produzieren.;;;;;;;;;;;x
 upgrade_crucible_of_storms;Crucible of Storms;Creuset des Temp�tes;Tiegel der St�rme;;;;;;;;;;;x
 upgrade_crucible_of_storms_desc;A long-submerged evil stirs beneath the Shrine of the Storm. The whispers that drive to madness echo within the depths, a siren song of eldritch power. Those who listen and obey prepare dark rituals to carry out their master�s will.;Un mal longtemps englouti s'agite sous le Sanctuaire des Temp�tes. Les murmures qui poussent � la folie r�sonnent dans les profondeurs, un chant de sir�ne de la puissance des anciens. Ceux qui �coutent et ob�issent pr�parent de sombres rituels pour ex�cuter la volont� de leur ma�tre.;Unter dem Heiligtum des Sturms r�hrt sich ein lange versunkenes �bel. In den Tiefen hallt das in den Wahnsinn treibende Fl�stern wider, ein Sirenengesang von �berirdischer Kraft. Diejenigen, die lauschen und gehorchen, bereiten dunkle Rituale vor, um den Willen ihres Meisters zu erf�llen.;;;;;;;;;;;x
+upgrade_queens_court;Queen's Court;;;;;;;;;;;;;x
```

Comment:
Ok, I'm pretty sure zumbak would object putting the new loc in the middle of the file again, but since it's ck2 and I can open the file with calc, I'm fine with it :P